vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,162 @@
+---
+title: vcpkg_make_configure
+description: Use vcpkg_make_configure to configure a Make-based project.
+ms.date: 01/30/2024
+---
+# vcpkg_make_configure
+
+Configure a Make-based project.
+
+## Usage
+        
+```cmake
+vcpkg_make_configure(
+    [AUTOCONFIG]
+    [COPY_SOURCE]
+    [DISABLE_MSVC_WRAPPERS]
+    [DISABLE_CPPFLAGS]
+    [DISABLE_DEFAULT_OPTIONS]
+    [DISABLE_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [SOURCE_PATH <source-path>]
```

Comment:
In usage instructions, square brackets usually means "optional". Is `SOURCE_PATH` optional?

```suggestion
    SOURCE_PATH <source-path>
```

Same question for the other parameters.